pkg/apputils/utils.go

```diff
@@ -23,6 +23,7 @@ import (
 type AppFactory interface {
 	GetKubeService() (kube.Kube, error)
 	GetAppService(ctx context.Context, name, namespace string) (app.AppService, error)
+	GetAppServiceForAdd(ctx context.Context, url, configUrl, namespace string, isHelmRepository bool, dryRun bool) (app.AppService, error)
```

Comment:
Shouldn't this be documented with the expected behaviour of implementations of the interface?

Reply:
We haven't been enforcing that on this project. I can add a doc comment to this `interface`, but the real solution will be to turn on the linting options for doc comments, then documenting everything at once.

Reply:
Documented